### PR TITLE
Mild QOL updates to vending machine-cargo interactions.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

This Pull makes a few minor improvements to the features I added with vendor cargo (#81582).
Changes are as follows:

* Vending machines, when restocked with a refill pack and while containing a number of credits, will play a cash register sound to alert you that there's a payout associated with the action.
* The Vending Restock app now works on all modular computer platforms, as opposed to just laptops and tablets.
    * This is actually hilarious because I even made a screen icon for modular consoles, and it's just been unused ever since merge.
* The "cargochat" cargo modular console now has the vendor refill app pre-installed.
* Vending machines, when determining their missing inventory during the associated station trait, now adds from 1-5 credits per missing item, up from 0-1. The original number was far too low to feel like a strong incentive based on feedback I'd received, and while still lower than providing a flat percentage of the missing item specifically (we ran some numbers on this a few months back), it should at least incentivize players to keep the station better stocked during regular rounds.

Also, the UI now shows a difference between if all vending machines are stocked or not.
image

## Why It's Good For The Game

Most of these are just interfacing and ease of use improvements, as having the app pre-installed in a place where players can actually make some use of it's contents should help to improve it's versatility with regards to the restocking app.

Audio feedback is always good.

In terms of the balance tweak on the stored value on vending machines, basically it's a way to better incentivize it as an option on the station and to better encourage players to do these kinds of minor upkeep tasks for the crew.

## Changelog

:cl:
qol: Vending machines now give audio feedback when you restock a vending refill and get a payout.
qol: The Restock tracker NTOS app for tracking vending machine contents now works on all consoles, and comes pre-installed on the cargochat cargo computers.
balance: Vending machines now offer a bit more credits when missing contents at the start of a round after getting restocked.
/:cl:

### CODE REVIEW

**Comment by ArcaneMusic** [comment]
I would say so, yeah
(I'll check later today)

> would it be bad if the app tracks the vending machines on the lavaland syndicate base?

I'd say so, yeah.

**Review by MrMelbert — APPROVED** [review]

**Comment by ArcaneMusic** [comment]
> > would it be bad if the app tracks the vending machines on the lavaland syndicate base?
> 
> I would say so, yeah (I'll check later today)

Confirmed that it does not show lavaland or free vending machine contents as there's no money to be collected inside.

Additionally, while I was in the ballpark, I made a tiny tweak to the TGUI so that when there's nothing to be replaced, the UI shows an all clear message:
image

File: tgui/packages/tgui/interfaces/RestockTracker.jsx [hunk 1]
```diff
@@ -81,7 +81,17 @@ export const RestockTracker = (props) => {
             </Stack.Item>
           </Stack>
         ))}
+        {vending_list.length ? null : <RestockTrackerFull />}
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/RestockTracker.jsx:84]:
```suggestion
        {vending_list.length === 0 && <RestockTrackerFull />}
```
explicit length check + no need to tell react to show null

**Comment by jlsnow301** [comment]
typescript? 🥺

**Pull request merged**